### PR TITLE
commands/object: use new cmds

### PR DESCRIPTION
**Pull request by overbool**
Refer: https://github.com/ipfs/go-ipfs/issues/5664
License: MIT
Signed-off-by: Overbool <overbool.xu@gmail.com>

### CODE REVIEW

**Comment by overbool** [comment]
@Stebalien CI checks have passed, could u help me review it?

File: core/commands/object/diff.go [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+	cmdenv "github.com/ipfs/go-ipfs/core/commands/cmdenv"
 	coreiface "github.com/ipfs/go-ipfs/core/coreapi/interface"
 	"github.com/ipfs/go-ipfs/dagutils"
 
+	cmds "gx/ipfs/QmSXUokcP4TJpFfqozT69AVAYRtzXVMUjzQVkYX41R9Svs/go-ipfs-cmds"
 	cmdkit "gx/ipfs/Qmde5VP1qUkyQXKCfmEUA7bP64V2HAptbJ7phuPp7jXWwg/go-ipfs-cmdkit"
 )
 
+const (
+	verboseOptionName = "verbose"
```
Comment (kevina) [inline core/commands/object/diff.go]: Is there a reason for this?

Nothing wrong with, but it's not very consistent as we don't use contacts for any of the other options in this file.

Personally, I would err on the side of not using the constant, but others may disagree.

Comment (overbool) [reply core/commands/object/diff.go]: @kevina `version` is used in two places, so I use the constant.

Comment (kevina) [reply core/commands/object/diff.go]: So is almost every other option, or am I missing something.

In any case it not that big of a deal.

Comment (Stebalien) [reply core/commands/object/diff.go]: Most of the commands have actually been changed to use constants for these.

**Pull request merged**